nasa/earthdata-search
# EDSC-4175: Add metric event to track users who are clicking through and selecting browse imagery

# Overview

### What is the feature?

Add metrics for the granule images modal and how it is used

### What is the Solution?

Adding metrics for granule image modal doing some cleanup of existing tests

### What areas of the application does this impact?

Granule thumbnail image modal 

# Testing

### Reproduction steps

- **Environment for testing:*Prod*
- **Collection to test with:*C2147480877-POCLOUD*

1. Any collection with granule thumbnails will do but, this one conveniently has multiple
2. Go to this collection, select a granule, ensure that events on google tag manager are firing for the following events
- Enlarge granule thumbnail viewer
- Download the granule
- Use the next and previous buttons on the granule thumbnail viewer
- Open the list view on the granule thumbnail viewer

### Attachments

Please include relevant screenshots or files that would be helpful in reviewing and verifying this change.

# Checklist
